pkg/apis/garden/types.go

```diff
@@ -623,8 +623,8 @@ type QuotaSpec struct {
 	ClusterLifetimeDays *int
 	// Metrics is a list of resources which will be put under constraints.
 	Metrics corev1.ResourceList
-	// Scope is the scope of the Quota object, either 'project' or 'secret'.
-	Scope QuotaScope
+	// ScopeReference is reference to an object version/kind.
+	ScopeReference corev1.ObjectReference
```

Comment:
Why do you name it differently in comparison to the field in the external version, i.e. `ScopeReference` instead of `Scope`?

Reply:
Very good question... I don't know :-D I renamed it back to `Scope`. PTAL.